spring-cloud-alibaba-nacos-config/src/main/java/com/alibaba/cloud/nacos/NacosConfigProperties.java

```diff
@@ -398,9 +383,50 @@ public Properties getConfigServiceProperties() {
 		else {
 			properties.put(ENDPOINT, endpoint);
 		}
+
+		enrichNacosProperties(properties);
 		return properties;
 	}
 
+	private void enrichNacosProperties(Properties properties) {
+		Map<String, Object> configurationItem = getConfigurationItemFromEnv(PREFIX);
+		configurationItem.forEach((k, v) -> {
+			if (!properties.contains(k)) {
+				properties.put(k, v);
+			}
+		});
+	}
+
+	private Map<String, Object> getConfigurationItemFromEnv(String prefix) {
+		Map<String, Object> configurationItems = new HashMap<>();
+		ConfigurableEnvironment configurableEnvironment = (ConfigurableEnvironment) environment;
+
+		MutablePropertySources propertySources = configurableEnvironment
+				.getPropertySources();
+		propertySources.stream().forEach(propertySource -> {
+			if (propertySource instanceof MapPropertySource) {
+				MapPropertySource mps = (MapPropertySource) propertySource;
+				mps.getSource().forEach((key, value) -> {
```

Comment:
The bootstrap defaultProperties PropertySource is `CompositePropertySource`, It can't get the expect value.

Reply:
Please review it again.